sbt/plan.sh

```diff
@@ -1,12 +1,12 @@
 pkg_origin=core
 pkg_name=sbt
-pkg_version=1.2.7
+pkg_version=1.2.8
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="A build tool for Scala, Java, and more"
 pkg_upstream_url="https://www.scala-sbt.org"
 pkg_license=("BSD-3-Clause")
```

Comment:
It looks like this is licensed as `Apache-2.0` according to https://github.com/sbt/sbt .  Could you verify that this is correct and update the license?

Reply:
Good god. You're right. I'm generally not checking the license on updates, perhaps that should become part of my routine. Good catch! Fixed.

Reply:
I haven't been either, but the similarity to strace made me pause for a second.  I'm going to be adding that to my checklist as well 😄